integration-tests/test/contracts.spec.ts

```diff
@@ -97,6 +100,22 @@ describe('Contract interactions', () => {
     let tokens: Contract[]
 
```

Comment:
What do you think about parsing the env as part of `OptimismEnv`? And also logging the values in the `before` block so we know they are being set

Reply:
I'm torn on this one. IMO `OptimismEnv` should contain stuff related to Optimism itself. Then each test can define its own env vars if they need additional ones. My worry about adding this to `OptimismEnv` is turning it into a "god class" that's difficult to maintain.

Agreed re: logging - I'll put that in.

Reply:
I see your point - I wouldn't be opposed to defining another getter class for the uniswap env vars to abstract away `process.env`, but fine as is also